packages/flutter_tools/test/integration.shard/test_utils.dart

```diff
@@ -19,33 +19,24 @@ Directory createResolvedTempDirectorySync(String prefix) {
   return globals.fs.directory(tempDirectory.resolveSymbolicLinksSync());
 }
 
-void writeFile(String path, String content) {
-  globals.fs.file(path)
+void writeFile(String path, String content, [bool sendToFuture = false]) {
+  final File file = globals.fs.file(path)
     ..createSync(recursive: true)
-    ..writeAsStringSync(content)
-    ..setLastModifiedSync(DateTime.now().add(const Duration(seconds: 10)));
-}
-
-void writePackages(String folder) {
-  writeFile(globals.fs.path.join(folder, '.packages'), '''
-test:${globals.fs.path.join(globals.fs.currentDirectory.path, 'lib')}/
-''');
-}
-
-void writePubspec(String folder) {
-  writeFile(globals.fs.path.join(folder, 'pubspec.yaml'), '''
-name: test
-dependencies:
-  flutter:
-    sdk: flutter
-''');
+    ..writeAsStringSync(content);
+  // When making edits for hot reload changes, the synchronous file write combined
```

Comment:
Make sure we don't do this when setting up the project, otherwise we may accidentally hot reload changes during the breakpoint test, causing them to be removed